bolt.write.time should actually be a counter, not a gauge (#22468)
hashicorp/vault: website/content/docs/upgrading/upgrade-to-1.14.x.mdx

## Important changes

`vault.raft_storage.bolt.write.time` has been corrected from a summary to a counter to more accurately reflect that it

> is measuring cumulative time writing, and not time spent per write.

Reviewer: ```suggestion
is measuring cumulative time writing, and not the distribution of individual write times.
```
